contracts/AssetPool.sol

```diff
@@ -132,23 +132,60 @@ contract AssetPool is Ownable, IAssetPool {
         address from,
         uint256 amount,
         address token,
-        bytes calldata
+        bytes calldata extraData
```

Comment:
Can we improve `@notice` documentation for `receiveApproval` to mention that an optional uint256 in `extraData` can be supplied and what it can be used for?

Reply:
Sure, done in 2a508a7